### PR TITLE
Mark contrastive test suit as "extra_large"

### PR DESCRIPTION
**Pull request by jbischof**
Part of #1738 

These are the slowest tests in our repo currently but this module is not in active development.

"extra_large" tests will be run manually when we push a new release

### CODE REVIEW

File: keras_cv/training/contrastive/contrastive_trainer_test.py [hunk 1]
```diff
@@ -24,6 +25,7 @@
 from keras_cv.training import ContrastiveTrainer
```
Comment (ianstenbit) [inline keras_cv/training/contrastive/contrastive_trainer_test.py:27]: Should we leave a comment here about why these are XL?
Maybe something like
"These tests are expensive and our contrastive training API is not yet released"?

File: cloudbuild/unit_test_jobs.jsonnet [hunk 2]
```diff
@@ -34,7 +34,7 @@ local unittest = base.BaseTest {
     |||
   ],
   command: [
-    'pytest --ignore keras_cv/models/legacy/',
+    'pytest --ignore keras_cv/models/legacy/ --durations 0',
```
Comment (ianstenbit) [inline cloudbuild/unit_test_jobs.jsonnet:37]: This is a nice feature -- didn't know this existed

**Review by ianstenbit — APPROVED** [review]
Thanks Jonathan!

File: cloudbuild/unit_test_jobs.jsonnet (hunk 2 above)
Comment (jbischof) [reply cloudbuild/unit_test_jobs.jsonnet:37]: Yes, and this way we can easily check in on slow tests from the GCB logs

**Pull request merged**